### PR TITLE
Updated Algorithm Binary Agents

### PR DESCRIPTION
**Pull request by atjonathan**
Refer #1094

### CODE REVIEW

File: Algorithm-Binary-Agents.md [hunk 1]
```diff
@@ -2,40 +2,54 @@
...
+
+### Problem Explanation:
 
 This problem is very straight forward, you will get a string that will represent a sentence in binary code, and you need to translate that into words. There is not direct way to do this so you will have to translate twice.
 
-## Hint: 1
+#### Relevant Links
+
+- [Link Title 1](http://example.com)
+- [Link Title 2](http://example.com)
```
Comment (atjonathan) [inline Algorithm-Binary-Agents.md]: Remember to add links....

File: Algorithm-Binary-Agents.md [hunk 2]
```diff
@@ -55,7 +69,13 @@ binaryAgent("01000001 01110010 01100101 01101110 00100111 01110100 00100000 0110
 - Convert to decimal by using parseInt(_binary_, 2) (with the second parameter we tell in which base our numbers currently are)
 - At the end, we return out converted message.
 
-## Code Solution 2:
+#### Relevant Links
+
+- [Link Title 1](http://example.com)
+- [Link Title 2](http://example.com)
```
Comment (atjonathan) [inline Algorithm-Binary-Agents.md]: Remember to add links....

File: Algorithm-Binary-Agents.md [hunk 3]
```diff
@@ -103,7 +123,13 @@ binaryAgent("01000001 01110010 01100101 01101110 00100111 01110100 00100000 0110
 - Convert the final decimal outside of the inner loop and then convert it to ASCII and saving it to **sentence** along with any other text string already converted and stored.
 - Reset the variable **decValue** to avoid getting wrong decimals before continuing to the outer loop.
 
-## Code Solution 3:
+#### Relevant Links
+
+- [Link Title 1](http://example.com)
+- [Link Title 2](http://example.com)
```
Comment (atjonathan) [inline Algorithm-Binary-Agents.md]: Remember to add links....

File: Algorithm-Binary-Agents.md [hunk 4]
```diff
@@ -123,32 +149,23 @@ binaryAgent("01000001 01110010 01100101 01101110 00100111 01110100 00100000 0110
...
-```javascript
-function binaryAgent(str) {
-  var re = /(\d+)(\s?)/g;
-  function convertToChar(match,p1,p2){
-    return String.fromCharCode(parseInt(p1, 2));
-  }
-  return str.replace(re, convertToChar);
-}
+- [Link Title 1](http://example.com)
+- [Link Title 2](http://example.com)
```
Comment (atjonathan) [inline Algorithm-Binary-Agents.md]: Remember to add links....

**Comment by Rafase282** [comment]
## General links

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/charCodeAt
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/fromCharCode

## Solution 1
https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/JS-String-Prototype-Split
https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/JS-ParseInt

## Solution 2
https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/JS-Math-Pow
https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/String.length

## Solution 3
https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/JS-Array-Prototype-Map

**Comment by Rafase282** [comment]
```diff
-- However `fromCharCode()` expects a series of numbers rather than an Array! We can use ES6 Spread Operator to pass in an Array of numbers as individual numbers. See here for more info; [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_operator)
+- However `fromCharCode()` expects a series of numbers rather than an Array! We can use ES6 Spread Operator to pass in an Array of numbers as individual numbers. See here for more info; [Spread operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_operator)
```

**Comment by Rafase282** [comment]
Or merge https://github.com/atjonathan/wiki/pull/3

**Comment by Rafase282** [comment]
# LGTM :+1:

**Pull request merged**